packages/core/core/telemetry/noop-tracer.ts

```diff
@@ -0,0 +1,69 @@
+import { Span, SpanContext, Tracer } from '@opentelemetry/api';
+
+/**
+ * Tracer implementation that does nothing (null object).
+ */
+export const noopTracer: Tracer = {
```

Comment:
What's this for?

Reply:
When the tracing is disabled, I use a `noopTracer` (null object pattern) so I don't have to change the main code in many places.

Reply:
I see. Probably ok then. The `noopTracer` just looks exactly like the `noopTracer` in `@opentelemetry/api`, but unfortunately I don't think it's exported.

Reply:
If it's exported that would be helpful (in case that's possible).